Code/RDGeneral/types.h

```diff
@@ -44,124 +44,123 @@ namespace RDKit {
+const int _QueryHCount           = 59;  // int
+const int _QueryIsotope          = 60;  // int
+const int _QueryMass             = 61;  // int = round(float * 1000)
+const int _ReactionDegreeChanged = 62;  // int (bool)
+const int NullBond               = 63;  // int (bool)
+const int _rgroupAtomMaps        = 64;  // int
+const int _rgroupBonds           = 65;  // int
 
 // SLN
-extern const std::string _AtomID; // unsigned int SLNParser
-extern const std::string _starred; // atom int COMPUTED (SLN)
-extern const std::string _SLN_s; // string SLNAttribs (chiral info)
-extern const std::string _Unfinished_SLN_; // int (bool)
+const int _AtomID          = 66;        // unsigned int SLNParser
+const int _starred         = 67;        // atom int COMPUTED (SLN)
+const int _SLN_s           = 68;        // string SLNAttribs (chiral info)
+const int _Unfinished_SLN_ = 69;        // int (bool)
 
 // Smarts Smiles
-extern const std::string _brokenChirality; // atom bool
-extern const std::string isImplicit;  // atom int (bool) 
-extern const std::string smilesSymbol; // atom string (only used in test?)
+const int _brokenChirality = 70;        // atom bool
+const int isImplicit       = 71;        // atom int (bool)
+const int smilesSymbol     = 72;        // atom string (only used in test?)
 
 // Tripos
-extern const std::string _TriposAtomType; // string Mol2FileParser
+const int _TriposAtomType = 73; // string Mol2FileParser
 // missing defs for _TriposAtomName//_TriposPartialCharge...
 
 
 ///////////////////////////////////////////////////////////////
 // misc props
-extern const std::string TWOD; // need THREED -> confusing using in TDTMol supplier
-                               //  converge with _2DConf?
-extern const std::string BalabanJ; // mol double
-extern const std::string BalanbanJ; // typo!! fix...
-
-extern const std::string Discrims; // FragCatalog Entry
-                                   // Subgraphs::DiscrimTuple (uint32,uint32,uint32)
-extern const std::string DistanceMatrix_Paths; // boost::shared_array<double>
-                                               //  - note, confusing creation of names in
-                                               //  - getDistanceMat
+const int TWOD                 = 74;    // need THREED -> confusing using in TDTMol supplier                                           //  converge with _2DConf?
+const int BalabanJ             = 75;    // mol double
+const int BalanbanJ            = 76;    // typo!! fix...
+const int Discrims             = 77;    // FragCatalog Entry
+                                        // Subgraphs::DiscrimTuple (uint32,uint32,uint32)
+const int DistanceMatrix_Paths = 78;    // boost::shared_array<double>
+                                        //  - note, confusing creation of names in
+                                        //  - getDistanceMat
+const int MAX                  = 78;   // reserved spaces
+const int RESERVED             = 256;
```

Comment:
This isn't used.  I was thinking that we could reserve some for the future, but MAX does the same thing really.